dev/TabView/TabView_themeresources.xaml

```diff
@@ -65,7 +69,7 @@
 
             <StaticResource x:Key="TabViewBorderBrush"                                      ResourceKey="CardStrokeColorDefault" />
             <StaticResource x:Key="TabViewItemBorderBrush"                                  ResourceKey="SubtleFillColorTransparentBrush" />
-            <Thickness x:Key="TabViewItemHeaderCloseButtonBorderThickness">0</Thickness>
+            <Thickness x:Key="TabViewButtonBorderThickness">0</Thickness>
```

Comment:
Unfortunately we can't remove the old resource, since that will break apps that may be depending on this resource. Since the old naming doesn't make sense, we can have both resources (and move the old one to a deprecated section).